askama_shared/src/generator.rs

```diff
@@ -811,6 +811,25 @@ impl<'a, S: std::hash::BuildHasher> Generator<'a, S> {
         buf.writeln(";")
     }
 
+    fn is_shadowing_variable(&self, var: &Target<'a>) -> bool {
```

Comment:
This is currently single use. Will you add more direct uses in the downstream PRs? If not, I'd prefer to just inline it.

Reply:
Yes, the method calls itself recursively for nested tuples and structs.

Reply:
Duh...